MultiServer.py

```diff
@@ -880,6 +901,8 @@ async def on_client_connected(ctx: Context, client: Client):
         'location_check_points': ctx.location_check_points,
         'datapackage_checksums': {game: game_data["checksum"] for game, game_data
                                   in ctx.gamespackage.items() if game in games and "checksum" in game_data},
+        "dynamic_checksums": {game: game_data["checksum"] for game, game_data in ctx.dynamic_package.items()
+                              if game in games and "checksum" in game_data},
```

Comment:
is it possible for a game to be in the dynamic_package and not have a checksum? i feel like that's something we want to error on instead of ignoring it.

Reply:
Yeah, all games  created now will have a checksum, so since this is a new feature it shouldn't check for deprecated data.